### PR TITLE
Fix adjust learning rate

### PR DESCRIPTION
**Pull request by Junonia**
Fix adjust learning rate, the original code does not work with the latest pytorch

### CODE REVIEW

**Comment by soumith** [comment]
thanks!
